src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -17,37 +17,89 @@ import ClimaCore.Operators as Operators
 # 1) turn the liquid_theta into theta version
 # 2) have a total energy version (primary goal)
 
-function eddy_diffusivity_coefficient(C_E::FT, norm_v_a, z_a, p) where {FT}
-    p_pbl = FT(85000)
-    p_strato = FT(10000)
-    K_E = C_E * norm_v_a * z_a
-    return p > p_pbl ? K_E : K_E * exp(-((p_pbl - p) / p_strato)^2)
+function compute_boundary_layer_height(Ri_c::FT) where {FT}
+    # Needs a column search given colidx
+    return nothing
+end
+function compute_bulk_richardson_number(
+    θ_v::FT,
+    θ_a,
+    norm_ua,
+    grav,
+    z,
+) where {FT}
+    norm²_ua = norm_ua^2# SurfaceFluxes uses FT(1) gustiness #TODO Add to ClimaParameters?
+    return (grav * z) * (θ_v - θ_a) / (θ_a * norm²_ua + eps(FT))
+end
+function compute_diffusion_coefficient(
+    Ri_a::FT,
+    Ri_c::FT,
+    zₐ::FT,
+    z₀::FT,
+    κ::FT,
+) where {FT}
+    # Equations (12), (13), (14)
```

Comment:
Add which paper? (Or better, add documentation, but I'm terrible at it so I won't force you to do that:)